### PR TITLE
Create Blog “identifying-the-best-agent-to-respond-in-your-ivr-system/index”

### PR DESCRIPTION
**Pull request by MichaelJolley**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/posts/identifying-the-best-agent-to-respond-in-your-ivr-system/index.md [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+
+```python
+async def _spanish_recording(response):
+   recording = "languages/spanish-recording.mp3"
+   spanish_transcript = await deepgram_transcribe(recording)
+
+   representative = customer_service_reps[spanish_transcript[1]]
+
+
+   response.say(f"This is the Spanish response and {representative} will help you.",
```
Comment (BekahHW) [inline content/blog/posts/identifying-the-best-agent-to-respond-in-your-ivr-system/index.md:154]: I don't think that 'f' should be there in the response.say

Comment (BekahHW) [reply content/blog/posts/identifying-the-best-agent-to-respond-in-your-ivr-system/index.md:154]: Also, maybe remove the period after you. It's probably grammatically correct, but it's distracting with the , after.

Comment (BekahHW) [reply content/blog/posts/identifying-the-best-agent-to-respond-in-your-ivr-system/index.md:154]: oh, maybe that f needs to be there.

File: content/blog/posts/identifying-the-best-agent-to-respond-in-your-ivr-system/index.md [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+
+
+   response.hangup()
+
+   return response
+```
+
+I also created a **templates** folder in the main Python Flask project directory with a blank **index.html** file. We don’t need anything in this file but feel free to add any HTML or Jinja.
+
+To run the application, I fired up two terminals simultaneously in Visual Studio Codel, one to run my Flask application and another for ngrok. Both are important, and you’ll need the ngrok url to add to your Twilio dashboard.
```
Comment (BekahHW) [inline content/blog/posts/identifying-the-best-agent-to-respond-in-your-ivr-system/index.md:204]: ** Visual Studio Code

**Review by BekahHW — CHANGES_REQUESTED** [review]

File: content/blog/posts/identifying-the-best-agent-to-respond-in-your-ivr-system/index.md (hunk 1 above)
Comment (geekchick) [reply content/blog/posts/identifying-the-best-agent-to-respond-in-your-ivr-system/index.md:154]: Yep, the f needs to be there. It's for string formatting. A weird Python thing lol

**Review by BekahHW — APPROVED** [review]

**Pull request closed without merging**